lang/src/org/partiql/lang/util/ExprValueFormatter.kt

```diff
@@ -49,6 +52,9 @@ class ConfigurableExprValueFormatter(private val config: Configuration) : ExprVa
 
                 DATE                                       -> out.append(value.scalar.dateValue().toString())
 
+                TIME                                       -> out.append("${ value.scalar.timeWithTimezoneValue()?.format(DateTimeFormatter.ofPattern("HH:mm:ss.nnnnnnnnnxxx")) 
+                                                                            ?: value.scalar.timeValue()?.format(DateTimeFormatter.ofPattern("HH:mm:ss.nnnnnnnnn")) }")
```

Comment:
This looks like something that should be in a utility function (or maybe that wrapper type I mentioned).

Reply:
Moved this logic to the `toString` method of the wrapper class.